### DISCUSSION TITLE
Does it support decimal conversion? Bin, hex and oct cannot be found when using

### DISCUSSION THREAD

**Discussion by DoraCoder** [josdejong/mathjs#2788] [Q&A]
The issues section is used only for bug reports. Please use the [Discussions](https://github.com/josdejong/mathjs/discussions) section to ask questions and share ideas and suggestions.

**Describe the bug**

**To Reproduce**
Steps to reproduce the behavior.

**Comment by josdejong** [comment]
Yes there are functions to parse and convert bin, hex, and oct values.

Can you elaborate what the problem is that you're facing?
